leaderboard.js

```diff
@@ -251,10 +268,20 @@ function emptyLeaderboard(msg) {
 
 }
 
+function findMember(message, id) {
+  return message.guild.members.cache.get(id);
+}
+
+function findUser(message, id) {
+  return findMember(message, id).user;
+}
+
 // Exports each function separately
 module.exports.addScore = addScore;
 module.exports.showLeaderboard = showLeaderboard;
 module.exports.position = position;
 module.exports.newChampionship = newChampionship;
 module.exports.dummyLeaderboard = dummyLeaderboard;
 module.exports.emptyLeaderboard = emptyLeaderboard;
+module.exports.findMember = findMember;
```

Comment:
Will these be used elsewhere, or are these functions internal to leaderboard.js?

Reply:
Removed. We can add them when we really would need them